### PR TITLE
Added info for WebXR

### PR DESCRIPTION
**Pull request by nicoledanuwidjaja**
Info about WebXR, WebXR device API, and technologies

### CODE REVIEW

File: docs/web-xr.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+WebXR is the open standard for running Virtual Reality and Augmented Reality experiences within a web browser. By unifying multiple technologies into a single truth, WebXR makes it easier to create immersive 3D cross-reality (XR) experiences. Such standards facilitate 3D rendering onto hardware for imaging virtual environments or graphical imagery.
+
+The WebXR feature set addresses challenges in maintaining data processing with low latency and high precision. WebXR Device APIs provide standardized interfaces with XR devices that recognize VR/AR-compatible devices, translate movement from input devices and sensors into controllable vectors, and render 3D scenes at appropriate frame rate to facilitate smooth interactions. The API is currently supported by Chrome and Firefox.
+
+The WebXR Device API provides easy spatial object interactions by adopting the "point and click" approach. Input sources are defined by pointer rays to designate input device locations to fire events in the virtual space.
+
+
+## Technologies
+WebXR is combined with the following frameworks and tools:
+- [A-frame](https://aframe.io/): web framework for building VR experiences with HTML and Entity-Component
```
Comment (avaer) [inline docs/web-xr.md:16]: This isn't technically correct -- A-Frame is not related to WebXR, other than that A-Frame uses WebXR as one API.

**Comment by avaer** [comment]
Will this automatically work when built with docusaurus?

**Comment by avaer** [comment]
@nicoledanuwidjaja Let me know when this should be reviewed again!

**Comment by nicoledanuwidjaja** [comment]
> @nicoledanuwidjaja Let me know when this should be reviewed again!

This is ready to go!

**Pull request merged**